packages/TorneloScoresheet/src/types/AppModeState.ts

```diff
@@ -41,6 +41,7 @@ export type TablePairingMode = {
 
 export type GraphicalRecordingMode = {
   mode: AppMode.GraphicalRecording;
+  startTime: GameTime;
```

Comment:
Yeh good point a game lasting more than 24 hours is not an actual issue, but what i just realised is if a game goes past midnight, wht you just said will be true!

Reply:
Ok fixed it, actually we only need to store the time in miliseconds for the start time, everywhere else we can stick to hours and minutes since we aren't comparing it to anything